recipes/fastkml/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "fastkml" %}
+{% set version = "0.11" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 86d0ee3b108a5508ab5b9ef1b7063fa6afd0115849bbb3f2d6c1e06cda5de279
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - pygeoif
+    - python
+    - python-dateutil
+
+test:
+  imports:
+    - fastkml
+  requires:
+    - pytest
+
+about:
+  home: "https://github.com/cleder/fastkml"
+  license: "GNU Library or Lesser General Public (LGPL)"
```

I think it should be
```suggestion
  license: "LGPL-2.1"
```